memory/ui/styles.css

```
@@ -5,451 +5,787 @@
 }
-[data-iii-ui="memory"] .mem-rail-caption {
-  font-size: 11px;
-  font-weight: 600;
-  text-transform: uppercase;
-  letter-spacing: 0.16em;
+[data-iii-ui="memory"] .mem-ui-banner .detail {
+  display: block;
+  font-family: var(--font-mono, ui-monospace, monospace);
+  font-size: 11.5px;
   color: var(--color-ink-faint);
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)